app/styles/main.scss

```diff
@@ -599,27 +612,21 @@ btn-mood label::before {
 }
 
 .progress-bar {
-	width: 100%;
```

Comment:
Ok!
(Sure that its not misaligned without its width and margins? I remember that @Jaapp-  fixed this before)